### ISSUE TITLE
ff_scoringhistory() and return fumbles for Sleeper

### ISSUE THREAD

**Issue by draftydesert277** [dynastyprocess/ffscrapr#281]
**Describe the bug**
Discovered when comparing aggregate ff_scoringhistory() results by team that ff_scoringhistory() appears to be ignoring return fumbles. This is a pretty obscure scoring thing that I think may be unique to Sleeper. See: https://support.sleeper.app/en/articles/4056849-why-did-my-player-lose-points-for-a-special-teams-fumble

**Reprex**
Please provide a short, reproducible example of the code you are trying to run. 
For more on reprexes, check out https://reprex.tidyverse.org

```r
 # Please include the code used for the connection object from `ff_connect()`
```
**Expected behavior**
Return fumbles to hit ff_scoringhistory() same as any other fumble

**Session information**
Please copy the output of `sessionInfo()` into the box below.
```r

```

**Screenshots**

**Additional context**
For reference, check out nflfastR data for Eagles/Rams, 2020. Cooper Kupp fumbled on a punt return, and I can verify that ff_scoringhistory() is 2.0 pts higher for my league than his official score in the Sleeper app.

**Comment by tanho63** [comment]
Hm! That’s interesting. Will try to see if nflfastr has return fumbles

**Comment by tanho63** [comment]
Hmm, it seems like they don’t have it, unfortunately: https://www.nflfastr.com/reference/calculate_player_stats.html

I’ll see if I can raise an issue and/or help fix it over there first and then can reopen this issue

**Comment by draftydesert277** [comment]
Good news - they do! I was actually able to pull the fumble I referenced specifically. Not sure how easy this is to incorporate into existing functionality but check this out:

lar_phi <- nflfastR::build_nflfastR_pbp(game_ids = "2020_02_LA_PHI")

lar_phi <- filter(lar_phi, fumbled_1_player_id == "00-0033908")

lar_phi_t <- transpose(lar_phi)

Also I admit this is a bit of a hunch as I wasn't able to easily verify this for other games.

**Comment by draftydesert277** [comment]
> Hmm, it seems like they don’t have it, unfortunately: https://www.nflfastr.com/reference/calculate_player_stats.html
> 
> I’ll see if I can raise an issue and/or help fix it over there first and then can revisit this issue

Oh! I see it's an issue with their stat aggregation. I assumed you were doing some wizardry with the raw pbp data.

I suppose I could work around this by creating a separate dataset of just kick/fumble returns based on build_nflfastR_pbp()?

**Comment by tanho63** [comment]
Yup, that’s definitely a way to get around it for now, I suspect that the nflfastr team would be happy to take in requests for these sort of aggregations

**Comment by tanho63** [comment]
Tacked on a request at nflverse/nflfastR#235, when that gets resolved I can re-map here.
